applications/DamApplication/python_scripts/check_and_prepare_model_process_dam_thermal.py

```diff
@@ -6,11 +6,16 @@ def Factory(settings, Model):
     return CheckAndPrepareModelProcessDamThermal(Model, settings["Parameters"])
 
 
-## All the processes python should be derived from "Process"
+## all the python processes should be derived from "python_process"
 class CheckAndPrepareModelProcessDamThermal(KratosMultiphysics.Process):
+    """Prepare the computing model part.
 
+    The computing model part is created if it does not exist. Nodes and elements
+    from the domain sub model parts are added to the computing model part.
+    Conditions are added from the processes sub model parts.
+    """
     def __init__(self, main_model_part, Parameters ):
-
+        KratosMultiphysics.Process.__init__(self)
```

Comment:
For the record, this was wrong before and could in some cases cause random segfaults
In the latest verdion pybind added a proper error, before it was hidden

Reply:
So, how should we proceed?

Reply:
Since it throws a meaningful error now, fix it if needed, just like you did here :)

Reply:
Ok, thank you for your comments and for approving :)